### DISCUSSION TITLE
Is there any way to notify all participants in a comment thread?

### DISCUSSION THREAD

**Discussion by JoeyZhouAus** [umputun/remark42#1288] [Q&A]
Hi guys,
i am wondering is there any way to configure the system or modify it to allow email notifications for all participants in a comment thread?
Many Thanks,
Joey

**Comment by paskal** [comment]
[Here](https://github.com/umputun/remark42/blob/c027dcd765238750d52db3e031c04b50288a3778/backend/app/notify/notify.go#L103-L129) is the logic of getting the recipient for the notifications you would need to alter for that.

**Comment by JoeyZhouAus** [comment]
Cool, Thanks @paskal
